docker/.env

```diff
@@ -0,0 +1,29 @@
+# host ip

```

Comment:
update the build-instructions doc to install the dependencies for the docker-compose

Reply:
ok